sqldelight-idea-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/inspections/UnusedImportInspection.kt

```diff
@@ -22,11 +22,17 @@ class UnusedImportInspection : LocalInspectionTool() {
     manager: InspectionManager,
     isOnTheFly: Boolean
   ): Array<ProblemDescriptor> {
-    val columnTypes = file.findChildrenOfType<SqlDelightJavaTypeName>()
-      .map { javaTypeName -> javaTypeName.text }
+    val sqlDelightColumnTypes = file.findChildrenOfType<SqlDelightColumnType>()
+    val columnTypes = sqlDelightColumnTypes.mapNotNull {
+      it.javaTypeName?.parameterizedJavaType?.firstChild?.text ?: it.javaTypeName?.text
+    }
+    val annotationTypes = sqlDelightColumnTypes.flatMap { it.annotationList }
```

Comment:
i think it would even cover the annotation types so this wouldn't be needeed

Reply:
yep